### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by enarhi**
### Issues addressed by this PR

Relates to #34 

Added readme.

### CODE REVIEW

**Comment by al-fisher** [comment]
Hey @enarhi - you may want to peep at the template in this admin issue 
https://github.com/BHoM/admin/issues/4 in case you want to include in this PR? 
Good timing! 😸

**Comment by enarhi** [comment]
@al-fisher perfect timing, updated as per that template and added you as a reviewer.

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# CarbonQueryDatabase
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/robot-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Robot_Toolkit/Robot_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=93&branchName=master)
```
Comment (FraserGreenroyd) [inline README.md:1]: References to `robot-toolkit` should be changed to reference this toolkit.

Comment (enarhi) [reply README.md]: Updated, the link to https://ci.appveyor.com/project/BHoMBot/CarbonQueryDatabase-Toolkit/branch/master appears to be a dead page unlike the Robot one, how do I get that page up and running for the build check on master?

Comment (FraserGreenroyd) [reply README.md]: As discussed @enarhi the page will be set up shortly so can go ahead and use it for now and accept the dead page :smile:

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
-# CarbonQueryDatabase
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/CarbonQueryDatabase-Toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/CarbonQueryDatabase_Toolkit/CarbonQueryDatabase_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build?definitionId=198&_a=summary)
```
Comment (FraserGreenroyd) [inline README.md:1]: I think the last link also needs to be updated to CarbonQueryDatabase_Toolkit

Comment (enarhi) [reply README.md:1]: I believe it already is as the link takes you to the CarbonQueryDatabase_Toolkit.CheckCore page. Please confirm!

Comment (FraserGreenroyd) [reply README.md:1]: You are correct - my mistake!

**Pull request merged**